### ISSUE TITLE
[i2c] sda interference during host enable

### ISSUE THREAD

**Issue by tjaychen** [lowRISC/opentitan#15131]
the current i2c assumes that when the host function is enabled, if `sda` is already low, then it is registered as bus interference.
@a-will pointed out that is not true in a multi-host situation. 

Since the current i2c [explicitly points out](https://docs.opentitan.org/hw/ip/i2c/doc/index.html#description) it does not support multi-host, this is not something that needs to be urgently fixed. 

Filing this away as something to be improved in the future when multi-host is added.

**Comment by tjaychen** [comment]
since this is actually only needed for multi-host, which we do not support, i'm kicking this to backlog.
Let me know if anyone objects.

**Comment by hcallahan-lowrisc** [comment]
Out of scope for D2 signoff, part of D3.

**Comment by vogelpi** [comment]
We should check if this is relevant for Earl Grey 2, I think it's not.

**Comment by vogelpi** [comment]
@hcallahan-lowrisc , would you mind checking if this still holds with the current version of the design please? I think you're the best person to look at this. If you disagree, please let me know :-)

**Comment by hcallahan-lowrisc** [comment]
This is no longer an issue now that multi-controller support has been merged. The bus_monitor initializes to a busy state upon enabling, and only asserts `bus_free` when SDA is seen to be inactive. `sda_interference` is gated on `controller_transmitting` - so while the controller is idle/waiting for `bus_free` it can't fire.
Hence closing.
